bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/ConfigDescriptionValidator.java

```diff
@@ -61,6 +65,10 @@ public static void validate(Map<String, Object> configurationParameters, URI con
         Preconditions.checkNotNull(configDescriptionURI, "Config description URI must not be null");
 
         ConfigDescription configDescription = getConfigDescription(configDescriptionURI);
+        if (configDescription == null) {
+            logger.warn("No config description could be found for the uri '{}'.", configDescriptionURI.toString());
+        }
+
         Map<String, ConfigDescriptionParameter> map = toMap(configDescription);
```

Comment:
I'll update the JavaDoc and I also just noticed that I missed adapting the tests - give me a few minutes...